### ISSUE TITLE
parse('*a, b = x') -> RuntimeError: type should be a Symbol not: s(:@ident

### ISSUE THREAD

**Issue by cben** [mvz/ripper_ruby_parser#3]
worked on ripper_ruby_parser 1.0.0:
```
irb(main):003:0> RipperRubyParser::Parser.new.parse('*a, b = x')
=> s(:masgn, s(:array, s(:splat, s(:lasgn, :a)), s(:__empty, s(:lvar, :b))), s(:to_ary, s(:call, nil, :x)))
```

Fails on 1.1.0:
```
RipperRubyParser::Parser.new.parse('*a, b = x')
RuntimeError: type should be a Symbol, not: s(:@ident, "b", s(1, 4))
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:242:in `process'
	from /home/bpaskinc/miq/grafresh/ripper_ruby_parser/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
	from /home/bpaskinc/miq/grafresh/ripper_ruby_parser/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:58:in `generic_add_star'
	from /home/bpaskinc/miq/grafresh/ripper_ruby_parser/lib/ripper_ruby_parser/sexp_handlers/assignment.rb:64:in `process_mlhs_add_star'
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
	from /home/bpaskinc/miq/grafresh/ripper_ruby_parser/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
	from /home/bpaskinc/miq/grafresh/ripper_ruby_parser/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:8:in `handle_potentially_typeless_sexp'
	from /home/bpaskinc/miq/grafresh/ripper_ruby_parser/lib/ripper_ruby_parser/sexp_handlers/assignment.rb:23:in `process_massign'
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
	from /home/bpaskinc/myenv/rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
... 14 levels...
```

The splat assignment syntax has been in ruby for a long time, e.g.
https://ruby-doc.org/core-2.0.0/doc/syntax/assignment_rdoc.html
> The * can appear anywhere on the right-hand side:  # RHS is typo, fixed now, but meant LHS
> `*a, b = 1, 2, 3`

**Comment by agrare** [comment]
Bisected to this commit https://github.com/mvz/ripper_ruby_parser/commit/1001ced065151455b19bcc4e11c59893fcd5201b

**Comment by mvz** [comment]
Thanks, @cben and @agrare. I've created a fix in #4.

**Comment by mvz** [comment]
I've released version 1.1.1 with the fix.
